CMakeLists.txt

```diff
@@ -86,6 +74,16 @@ if(BUILD_SHARED_LIBS)
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
   )
 
+  if(BUILD_TSS2)
+    target_link_libraries(xaptum-tpm PUBLIC
+      tss2-sys
```

Comment:
Can the same `tss2::sys`  syntax be used in both cases?   Maybe just add an `tss2::sys` alias in our `CMakeLists.txt`, same as is in the `FindTSS2.cmake` script.

Reply:
Ha, I actually originally did have that (the library alias), but worried that it actually made the static library case too messy

I just pushed a variant with this change

Reply:
Sorry @drbild I didn't notice a naming problem that made the CI fail. Fixed now